.github/workflows/lint-and-test.yml

```
@@ -0,0 +1,114 @@
+name: lint and test
+
+on: [push, pull_request]
+
+jobs:
+    build-conode:
+        name: build conode
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: conode
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-go@v2
+
+            - run: make docker_start_detached docker_wait
+            - run: docker exec conode sh -c './bcadmin latest --bc conodes/bc*cfg'
+
+    test-dynacred:
+        name: test dynacred
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                working-directory: dynacred
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+
+            - uses: actions/checkout@v2
+              with:
+                  repository: dedis/cothority
+                  path: cothority
+            - run: make docker
+              working-directory: cothority
```

```suggestion
            - run: ( cd cothority && make docker )
```

Or is that too cryptic?

Again: why do you prefer to have multiple `-run:` here instead of a `-run: |`?

Same question for the other steps.

---

IMO, using `working-directory` makes the job more readable, by removing the `cd dynacred &&` for each step.